packages/cli/src/commands/network/whitelist-l2.test.ts

```diff
@@ -0,0 +1,30 @@
+import { newKitFromWeb3 } from '@celo/contractkit'
+import { testWithAnvil } from '@celo/dev-utils/lib/anvil-test'
+import Web3 from 'web3'
+import { setupL2 } from '../../test-utils/chain-setup'
+import { testLocallyWithWeb3Node } from '../../test-utils/cliUtils'
+import Whitelist from './whitelist'
+
+process.env.NO_SYNCCHECK = 'true'
+
+const spy = jest.spyOn(console, 'log')
+
+afterAll(() => {
+  jest.clearAllMocks()
+})
+
+testWithAnvil('network:whitelist cmd', (web3: Web3) => {
+  test('can print the whitelist', async () => {
+    const kit = newKitFromWeb3(web3)
+
+    await setupL2(kit)
+
+    await testLocallyWithWeb3Node(Whitelist, [], web3)
+    expect(spy.mock.calls[0][0]).toMatchInlineSnapshot(`
+      "Available currencies:
+      0x2A3733dBc31980f02b12135C809b5da33BF3a1e9 - Celo Brazilian Real (Celo Brazilian Real)
```

Comment:
its odd that the name shows twice instead of the symbol

Reply:
that's because both symbol and name for the anvil devchain return the same value:

```bash
~ cast call 0x2A3733dBc31980f02b12135C809b5da33BF3a1e9 "symbol()(string)" --rpc-url 127.0.0.1:8546

"Celo Brazilian Real"

~ cast call 0x2A3733dBc31980f02b12135C809b5da33BF3a1e9 "name()(string)" --rpc-url 127.0.0.1:8546

"Celo Brazilian Real"